portfolio/src/main/webapp/js/script.js

```diff
@@ -78,7 +78,8 @@ const responsabilities = [
 ]
 
 const ENTER_KEY = 13;
-
+const RIGHT_CLICK = 2;
+let nQuestionsToLoad = 2;
```

Comment:
As much as I like Hungarian notation, Google's style guide says to avoid it. Change it to something like "questionLoadCount"?

Reply:
Of course! Thanks for the feedback